### PR TITLE
Update to v6.6.50

### PR DESCRIPTION
**Pull request by DaftB0t**
Sync with upstream v6.6.50

### CODE REVIEW

**Review by randytsi369 — DISMISSED** [review]

**Comment by ts-kris** [comment]
Blocked until #128 is merged, which is waiting on hardware for CI rack

**Review by markfeathers — DISMISSED** [review]

**Review by markfeathers — DISMISSED** [review]

**Pull request closed without merging**
